app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,70 @@
+class PasswordResetsController < ApplicationController
+  PASSWORD_RESET_PERMIT = %i(
+    password
+    password_confirmation
+  ).freeze
+
+  before_action :load_user, :valid_user,
+                :check_expiration, only: %i(edit update)
+
+  # password_resets POST (/:locale)/password_resets
+  def new; end
+
+  def create
+    @user = User.find_by email: params.dig(:password_reset, :email)&.downcase
+
+    if @user
+      @user.create_reset_digest
+      @user.send_password_reset_email
+      flash[:info] = t(".notice")
+      redirect_to root_url
+    else
+      flash.now[:danger] = t(".failure")
+      render :new, status: :unprocessable_entity
+    end
+  end
+
+  # edit_password_reset GET (/:locale)/password_resets/:id/edit(.:format)
```

Comment:
```suggestion
  # GET (/:locale)/password_resets/:id/edit(.:format)
```
key ở đầu thừa (edit_password_reset), fix tương tự những chỗ khác